lib/enum/arrays.rb

```diff
@@ -1,24 +1,37 @@
 class Array
 
+  def initialize(array)
+    @array = array
+  end
+
   def sum_of_numeric_elements
+    @array.select {|num| num.class == Fixnum || num.class == Float}.inject(:+)
```

Comment:
Yes, and you can also get both Fixnums and Floats in one go with Numeric, like this: `num.class == Numeric` or `num.is_a? Numeric`